Clinical-Genomics/scout
# Display observations

This PR should fix #1274 

**How to test**:
1. Make sure scout stage has installed the current master branch.
1. Go to this variant : https://scout-stage.scilifelab.se/cust002/F0027817/aba7ac625e06534d71ed16e3766ffc59 
and make sure that the local observations table looks like this:
![image](https://user-images.githubusercontent.com/28093618/62473239-a573f080-b7a0-11e9-9bfa-10015648376f.png)

1. install this branch on clinical-db stage using this command:  
```
bash update-scout-stage.sh display_observations
```

Go to the link above and make sure that now the page displays a link to another case's variant (the other observation) and the link works and points to the other variant:

![image](https://user-images.githubusercontent.com/28093618/62473071-529a3900-b7a0-11e9-9c3c-073569632d58.png)


**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
